blockstack/lib/nameset/virtualchain_hooks.py

```diff
@@ -571,8 +573,7 @@ def db_save( block_height, consensus_hash, ops_hash, accepted_ops, virtualchain_
        log.error("FATAL: no state engine given")
        os.abort()
 
-
-def db_continue( block_id, consensus_hash ):
+def db_continue( block_id, consensus_hash, db_state=None ):
```

Comment:
This `db_continue` function appears to be the closest thing to an "end-of-block" hook. However, it requires the `db_state` arg, which requires a small patch to the `virtualchain` dependency lib in another repo. Does this all seem right?

Reply:
It should be safe to do all of this at the end of `db_save()`.  Then you won't have to patch `virtualchain`.

Reply:
Great, was able to move logic to that function and works fine 👍